txpool/txpool.go

```diff
@@ -841,14 +842,17 @@ func (t *txPriceHeap) Length() uint64 {
 	return uint64(len(t.index))
 }
 
-func (t *txPriceHeap) Delete(tx *types.Transaction) {
+func (t *txPriceHeap) Delete(tx *types.Transaction) *types.Transaction {
```

Comment:
Maybe we should return a boolean value here instead of transaction

Reply:
[Changed](https://github.com/0xPolygon/polygon-sdk/pull/251/commits/5aa22e1de9f29b6cb1dded78f184a018649569c6)